modules/monitoring-determining-why-prometheus-is-consuming-disk-space.adoc

```diff
@@ -13,6 +13,9 @@ Every assigned key-value pair has a unique time series. The use of many unbound
 You can use the following measures when Prometheus consumes a lot of disk:
 
 * *Check the number of scrape samples* that are being collected.
+
+* *Check the TSDB status in the Prometheus UI* for more information on which labels are creating the most time series. This requires cluster administrator privileges.
```

Comment:
Just wondering, are people supposed to know what TSDB is, or should it be spelled out? I don't, but that doesn't mean much ;)

Reply:
Good question! When I researched it, I only found it referred to as TSDB. But I was just able to find a white paper with it defined as "time series database," so I can spell it out. 👍